src/Microsoft.PowerShell.Commands.Management/AssemblyInfo.cs

```diff
@@ -1,8 +1,5 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyFileVersionAttribute("3.0.0.0")]
-[assembly: AssemblyVersion("3.0.0.0")]
-
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en-US")]
```

Comment:
Same here and in other files. So for this one, the AssemlbyInfo.cs file can be deleted probably.